### DISCUSSION TITLE
Localized Navigation with strapi 3.6

### DISCUSSION THREAD

**Discussion by prKrumm** [VirtusLab/strapi-plugin-navigation#60] [Ideas]
Any ideas how this plugin can be used with strapi 3.6 and its new i18n feature? 

If I want to associate a localized page with a navigation item I can pick only pages of the default locale.

**Comment by cyp3rius** [comment]
Good point @prKrumm we have got some ideas but first of all we want to collect your feedback and see which approach is the most accurate :)

Converting to discussion

**Comment by cyp3rius** [comment]
Possible options:

1. Localized navigation that got access to the content in a same language only and non-localized content types.
2. Multiple navigation structures with possibility to pre-filter by any localization if applicable 
3. As now but for localized content types there should be localization selector

How the above sounds?

**Reply by jfahrni** [reply]
Hi @cyp3rius 
Thank you for your Navigations-Plugin!

Since we will soon finish the first multilingual project with Strapi, we are in need of a navigation that is also usable in a multilingual environment.

To answer your Question above: we believe that number 1 would fit best for us.

- Only one Navigation for all Languages
- The Navigation Items should be translatable (Title / URL)  but not the Settings (Attach to Menu / Type / ContentType / Entity)
- The content is shown in the same language, or if not available in the default Language

We are curious about the further development

Best, Jonas

**Reply by haZya** [reply]
If this is still up for debate I would suggest option 2. The reason being that option 1 seems to limit the customizability of the overall navigation structure for each language.

For instance, an application may have a different navigation structures for different languages where some pages may not exist for another language. If we use only one navigation for all languages I don't know if there is any good way to achieve this. If we go with option 2 and have a separate navigation for each language, we could easily have different structure for each language if needed. 

Also option 2 is used by other CMSs like WP so it seems to me like the standard choice.

Any thoughts?

**Reply by Krislunde** [reply]
Just want to pitch my $.02 in and say that option 2 does indeed sound like the most flexible option. Rarely do localized versions of full organisational websites have the exact same site structure/navigation.

**Reply by wanbinkimoon** [reply]
up vote option 2

**Comment by cyp3rius** [comment]
Closing as there is no longer support for `v3`
